x-pack/platform/plugins/private/translations/translations/fr-FR.json

```diff
@@ -5449,8 +5449,6 @@
     "kbn-esql-validation-autocomplete.esql.autocomplete.examplesLabel": "Exemples :",
     "kbn-esql-validation-autocomplete.esql.autocomplete.fieldDefinition": "Champ spécifié par le tableau d'entrée",
     "kbn-esql-validation-autocomplete.esql.autocomplete.integrationDefinition": "Intégration",
-    "kbn-esql-validation-autocomplete.esql.autocomplete.specialIndexes.indexType.alias": "Alias",
```

Comment:
Q: Why are the translations changes needed? Or is it a lint rule thing to reorder them?

Reply:
As I changed some translations I run `node scripts/i18n_check --fix` and that was the result.